src/react/utils/storageOptions.js

```diff
@@ -48,11 +49,14 @@ export function getLocationBucketName(locationConstraint, locations) {
 export function selectStorageOptions(
     capabilities: $PropertyType<InstanceStateSnapshot, 'capabilities'>,
     labelFn?: LabelFunction,
+    exceptOldVersion: boolean = true,
 ): Array<StorageOptionSelect> {
     return Object.keys(storageOptions).filter(o => {
-        const isOldVersion = !!storageOptions[o].isOldVersion;
-        if (isOldVersion) {
-            return;
+        if (exceptOldVersion) {
```

Comment:
Let's return a boolean from the filter callback.

Reply:
> 
> 
> @Cuervino @terrydervaux What should we do if the user has already created a "now hidden" location (e.g. NFS Mount location)? Should user be able to see it / edit it / delete it?
> @theocerutti "legacy support" tests would be needed.

This case will not happen outside of our test environment, no?